.travis.yml

```
@@ -10,15 +10,10 @@ env:
 
 matrix:
   include:
-    - env: RUNTIME=2.7
-    - env: RUNTIME=3.5
     - env: RUNTIME=3.6
-    - os: osx
-      env: RUNTIME=2.7
-    - os: osx
-      env: RUNTIME=3.5
     - os: osx
       env: RUNTIME=3.6
+    fast_finish: true
```

```suggestion
    - os: osx
      env: RUNTIME=3.6
  fast_finish: true
```

looks like `fast_finish` needs to be at the `matrix` level, not `include` level.